Lushu/src/test/kotlin/lushu/Lushu/LogGenerator/LogGenerator.kt

```diff
@@ -104,7 +134,26 @@ class LogGenerator(
                 .replace("{ACTION}", action)
                 .replace("{DATETIME}", datefmt.format(Date()))
 
-            println(message)
+            System.out.println(message)
```

Comment:
```suggestion
            println(message)
```
I believe `println` works just as well as `System.out.println`. From my tests it was being intercepted by the custom PrintStream